zcash_client_backend/src/constants/regtest.rs

```diff
@@ -0,0 +1,38 @@
+/// The testnet coin type for ZEC, as defined by [SLIP 44].
```

Comment:
SLIP 44 does not define a cointype for regtest, because regtest is a `zcashd`-specific local testing environment. Alter this to instead say that `zcashd` reuses the testnet coin type for its regtest mode.

Also add a module comment indicating that these constants are all specific to `zcashd`'s local regtest mode.

Reply:
Done